website/js/VariantTable.js

```diff
@@ -180,8 +180,8 @@ const researchModeGroups = [
                     {title: 'EAS Allele frequency (ExAC minus TCGA)', prop: 'Allele_frequency_EAS_ExAC', core: true},
                     {title: 'FIN Allele frequency (ExAC minus TCGA)', prop: 'Allele_frequency_FIN_ExAC', core: true},
                     {title: 'NFE Allele frequency (ExAC minus TCGA)', prop: 'Allele_frequency_NFE_ExAC', core: true},
-                    {title: 'OTH Allele frequency (ExAC minus TCGA)', prop: 'Allele_frequency_OTH_ExAC', core: true},
-                    {title: 'SAS Allele frequency (ExAC minus TCGA)', prop: 'Allele_frequency_SAS_ExAC', core: true}
+                    {title: 'SAS Allele frequency (ExAC minus TCGA)', prop: 'Allele_frequency_SAS_ExAC', core: true},
+                    {title: 'OTH Allele frequency (ExAC minus TCGA)', prop: 'Allele_frequency_OTH_ExAC', core: true}
```

Comment:
was this a requested change?

Reply:
It wasn't a requested change, no. It was one of those cases where I was looking at the code, the help text didn't match up (it has "Other" at the end), and it didn't make sense to me to have "Southeast Asian" at the end either, so I quickly edited it. Is it more appropriate to remove it here and file it as an issue instead?